packages/query/src/graphql/plugins/logGraphql.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {performance} from 'perf_hooks';
+import {ApolloServerPlugin} from 'apollo-server-plugin-base';
+import {GraphQLRequestListener} from 'apollo-server-plugin-base/src/index';
+import {GraphQLRequestContext} from 'apollo-server-types';
+import {getLogger} from '../../utils/logger';
+
+const logger = getLogger('graphql');
+
+const getSizeInBytes = (obj: Record<string, unknown>) => {
+  let str: string;
+  if (typeof obj === 'string') {
```

Comment:
the obj is Record type, is it still possible to be string?

Reply:
made a mistake here, forgot to update.